<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_tableView_BD_or_XML">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_Display_BD_or_XML"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbDisplayBDtable">
        <property name="text">
         <string>Отобразить данные из БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAddRow">
        <property name="text">
         <string>Добавить строку в таблице БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRemoveRow">
        <property name="text">
         <string>Удалить выделенную строку в таблице БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGenerationXML">
        <property name="text">
         <string>Сгенерировать XML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbReadXml">
        <property name="text">
         <string>Открыть XML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
